pkgs/development/haskell-modules/configuration-common.nix

```
@@ -2464,6 +2464,15 @@ self: super: {
   # 2022-03-16: Upstream stopped updating bounds https://github.com/haskell-hvr/base-noprelude/pull/15
   base-noprelude = doJailbreak super.base-noprelude;
 
+  # 2024-01-14: Support hashable 1.5 https://github.com/kowainik/relude/pull/463/
```

We still have hashable 1.4.7.0 :)

```suggestion
  # 2024-01-14: Support hedgehog 1.5 https://github.com/kowainik/relude/pull/463/
```

---

Derp